### ISSUE TITLE
Add 'wheelchair accessibiltiy'

### ISSUE THREAD

**Issue by CloCkWeRX** [osmlab/onosm.org#47]
https://github.com/osmlab/onosm.org/pull/2/commits/4b3e8214a0f72e1350c4f26d5c26bc963d6acca8 but internationalized

**Comment by le-jun** [comment]
It has been 4 years already and I am unsure about it, what are we still waiting for to add wheelchair accessibility ?

**Comment by iandees** [comment]
I'm happy to review a pull request if you have something in mind.

**Comment by le-jun** [comment]
Don’t get me wrong, I am simply asking from a non-expert point of view what the first message mean. I understand that it would add a Wheelchair section in the form, but what does "internationalized" mean ? Is it simply a matter of translating the content of it ?

**Comment by iandees** [comment]
Yep, it would be deciding what questions to ask and then adding the fields to the form. Then posting here to ask folks for help with localization.

**Comment by le-jun** [comment]
According to the [wiki](https://wiki.openstreetmap.org/wiki/Key:wheelchair), which cite the Americans with Disabilities Act, the question to ask would be like: "Is your business accessible to wheelchair users?" followed by three choices:

* Yes: No steps
* Limited: Entrance step only, lower than 7 cm / 3 inch​
* No: Steps higher than 7 cm / 3 inch​

Another solution would be to make use of the wheelchair:description key with a free form so that users can type freely the same way as [Wheelmap](https://wiki.openstreetmap.org/wiki/Wheelmap).

To go one step further, we could also provide a checkbox for the [ramp](https://wiki.openstreetmap.org/wiki/Key:ramp) key that would add a "ramp=yes" attribute to the note. The ramp:wheelchair key seems a bit too much to ask as it requires the incline of the ramp.

**Comment by opyh** [comment]
Hi, I'm one of the maintainers of Wheelmap 👋 First, thanks for your awesome work on this project!

# tl/dr

- Use `wheelchair=yes/no/limited`, re-use Wheelmap's [localization](https://github.com/sozialhelden/wheelmap-frontend/tree/main/public/i18n) if you want
- Add `wheelchair:description`, but don't use it to replace other tags
- Optionally add `ramp:wheelchair` if possible, but be sure that it's used correctly
- Get inspired by popular tags combined with `highway=steps` and `entrance` 

# Explanation

## `wheelchair=yes/no/limited`

…is a very good idea, this is what Wheelmap uses. Yes/no/limited can be either deduced, or you could explicitly ask about the specific requirements that @le-jun listed, in separate questions. So far, Wheelmap has had good experiences with this 'traffic light system'. Feel free to get inspired by our UI and even copy our localization: 
<img alt="UI of Wheelmap, asking about the wheelchair accessibility of a playground" src="https://user-images.githubusercontent.com/192912/128837954-ad1e897c-bb20-4bc0-bc82-23fa0e320529.png">

Localization files: https://github.com/sozialhelden/wheelmap-frontend/tree/main/public/i18n

## `wheelchair:description`

Often containing very valuable information for complex, difficult to tag issues though, like: 'You have to ring some bell and then use a second special entrance via a courtyard and an elevator'. This is information that you can't convey with tags, so I'd absolutely add it, too. Important: It should *not* be used to replace other tags, though, because:

- It *is* helpful for finding out about a specific place you know already ('can I enter the cafe that my friends go to?'), but not for place discovery ('where can I go at all?')
- Its language is often not defined, and OSM clients have to auto-translate the field for their users before its usable for international tourism cases

## `ramp:wheelchair`

+1 for `ramp:wheelchair`! It needs a warning or additional question though, to ensure that the owner knows that the incline is really <4%, otherwise its unusable for most manual wheelchair users, and even dangerous in winters. Don't forget that the following thing is considered a ramp, too:

![A photo of a very steep stroller ramp built into some stairs](https://user-images.githubusercontent.com/192912/128837600-dc453c4f-415c-43e0-8a83-af3be3697ee0.jpg)

## More infos about entrances

Extra bonus points for this: The number and height of steps at the entrance is very valuable information, too. Many wheelchair users know exactly how high they can climb, and people with mobility impairments who use no wheelchair can use it to decide if they can or cannot enter a place. For OSM mapping, it's not sensible to add it to a place directly, but to its entrance. Inspiration: [`entrance`](https://wiki.openstreetmap.org/wiki/Key:entrance), [highway:steps](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dsteps)

# Why is this so important?

- more a11y tags mean more discoverable parts of the world for people with disabilities
- tags help cities to calculate accurate budgets to spend money on accessibility
- tags help NGOs raise awareness about local accessibility issues. Data often makes for a good argument
- a11y tags allow apps to rank places by their accessibility, making accessibility a key marketing issue for places. This might move money towards accessibility budgets.

Ultimately, tagging can make the real world itself more accessible for everybody.

**Comment by iandees** [comment]
This has already been implemented — the form has a "wheelchair accessibility" field (`step2.wheel`) with Yes/Limited/No/Unknown options mapping to `wheelchair=yes/limited/no`, plus an explanatory modal describing each option, matching the approach the Wheelmap folks recommended above. It's translated in every locale. Closing as done.
